GooglePlayInstant/Editor/GenericLoadingScreenScript.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.Networking;
+
+public class GenericLoadingScreenScript : MonoBehaviour {
+    private AssetBundle bundle;
+    IEnumerator Start () 
+    {
+        yield return StartCoroutine(GetAssetBundle());
+        GoToGame();
+    }
+
+    IEnumerator GetAssetBundle() 
+    {
+        UnityWebRequest www = UnityWebRequestAssetBundle.GetAssetBundle("__ASSETBUNDLEURL__");
```

Comment:
I know __ASSETBUNDLEURL__ is a placeholder, can you add a TODO here to not hardcode it. Also, is there a placeholder that we could put here that actually works so that anyone who syncs to this commit could test things out end-to-end?

Reply:
Just as an alternative to consider for the string placeholder:
Could you put all of the config in a config file that is then parsed at runtime? E.g. json?
{ "main_scene_asset_bundle_url": "https://mycdn.com/maingame" }

If you do this, do you need to copy the files from the Editor or can they just live one directory up under GooglePlayInstant for direct use by the Engine?

Is a goal to avoid including engine code that may not be used by some developers?

Reply:
This config file could not be in the GooglePlayInstant directory because it must be accessible in the project at the time the instant app is started on a device (so something Unity would build ultimately). I could place this config file in the same directory as the new LoadingSceneScript file. Would that option be preferable? I feel like it would still be very similar to what's going on currently, except with more reading done in the LoadingSceneScript file for everyone person who uses the instant app.

Reply:
I think it would be cleaner. The shipping plugin would have:
 - LoadingSceneScript in the engine directory that contains code to parse a play-instant-config.json file
 - Editor code that knows how to create a play-instant-config.json
 - No play-instant-config.json file anywhere (we don't include a default file)
 - No GenericLoadingSceneScript in the editor directory

After the developer clicks the "create loading scene" button (we might rename it to "configure loading scene") then we create the play-instant-config.json at a well-defined location.

Let me know if this seems reasonable or not.